helmfile/charts/networkpolicy/generator/values.yaml

```
@@ -0,0 +1,115 @@
+fullnameOverride: ""
+nameOverride: ""
+
+# This helm chart allows for network policies to be generated with a much simpler definition and with the option for reuse of rules
+
+# The rules can be imported into polices by specifying the name under the "rule" key within the policy
+rules:
+  # Rule format
+  # <rule-name>:
+  #   peers:
+  #     # with IPs
+  #     - cidr: 0.0.0.0/0
+  #     # with labels
+  #     - namespace: <namespace>
+  #       labels:
+  #         <key>: <value>
+  #   ports:
+  #     - <protocol>: <port>
+
+  # egressRuleDNS:
+  #   ports:
+  #     - tcp: 53
+  #     - udp: 53
+
+  # egressRuleApiServer:
+  #   peers:
+  #     - cidr: 0.0.0.0/0
+  #   ports:
+  #     - tcp: 6443
+
+  # egressRuleIngressSC:
+  #   peers:
+  #     - cidr: 0.0.0.0/0
+  #   ports:
+  #     - tcp: 443
+
+  # egressRuleIngressWC:
+  #   peers:
+  #     - cidr: 0.0.0.0/0
+  #   ports:
+  #     - tcp: 443
+
+  # egressRuleObjectStore:
+  #   peers:
+  #     - cidr: 0.0.0.0/0
+  #   ports:
+  #     - tcp: 443
+
+  # ingressRuleApiServer:
+  #   peers:
+  #     - cidr: 0.0.0.0/0
+
+  # ingressRuleBlackbox:
+  #   peers:
+  #     - namespace: monitoring
+  #       labels:
+  #         app.kubernetes.io/name: blackbox
+
+  # ingressRulePrometheus:
+  #   peers:
+  #     - namespace: monitoring
+  #       labels:
+  #         app.kubernetes.io/name: prometheus
+
+# The policies are organised under namespaces: "policies.<policy-namespace>.<policy-name>"
+policies:
+  # Policy format
+  # <policy-namespace>:
+  #   <policy-name>:
+  #     labels:
```

For clarity this migh want to be

```suggestion
  #     podSelectorLabels:
```